tests/acceptance/29_audit_mode/audit_safe_functions.cf

```diff
@@ -0,0 +1,16 @@
+bundle agent __main__
+{
+  vars:
+    "cf_agent" string => ifelse(isvariable("sys.cf_agent"), "$(sys.cf_agent)", "/var/cfengine/bin/cf-agent");
+#    "result" string => execresult("$(cf_agent) --audit -Kf $(this.promise_filename).sub", useshell);
```

Comment:
It will be `--audit=manifest` or `--audit=diff`.

Reply:
ack